src/EventListener/CartClearedSubscriber.php

```diff
@@ -55,9 +56,12 @@ public function addScriptWhenCartEmpty(): void
             return;
         }
 
-        $this->tagBag->add(new TwigTag('@SetonoSyliusAddwishPlugin/Tag/cart_cleared.js.twig', TagInterface::TYPE_SCRIPT, [
-            'cart' => $cart,
-        ]), TagBagInterface::SECTION_BODY_BEGIN);
+        $this->tagBag->add(new TwigTag(
+            '@SetonoSyliusAddwishPlugin/Tag/cart_cleared.js.twig',
+            TagInterface::TYPE_SCRIPT,
+            Tags::TAG_CART_CLEARED,
+            ['cart' => $cart]
+        ), TagBagInterface::SECTION_BODY_BEGIN);
```

Comment:
Should be at the `TagBagInterface::SECTION_BODY_END` I guess.

Reply:
Yes!